website/templates/website/base_frame.html

```diff
@@ -28,4 +28,5 @@
 {% block footer %}{% endblock %}
 </body>
 {% block extra_js_bottom %}{% endblock %}
+{{ google_analytics|safe }}
```

Comment:
yes it could have been hardcoded. But in this way it will be very easy to change the Tracking ID from settings.py. So we do not have to  copy the snippet again and again in all templates.